spec/mediators/translation_update_mediator_spec.rb

```diff
@@ -331,4 +338,57 @@
       expect(translation.reviewer).to be_nil
     end
   end
+
+  describe '#check_and_invoke_article_pinger' do
+    let(:article_webhook_url) { 'https://webhooks.com/translation_ready' }
+    let(:project) { FactoryBot.create(:project, targeted_rfc5646_locales: { 'fr' => true, 'fr-CA' =>true, 'fr-FR' => true }, article_webhook_url: article_webhook_url) }
+    let(:article_ready) { true }
+    let(:article) { FactoryBot.create(:article, project: project, targeted_rfc5646_locales: {'fr-CA' => true}, ready: false ) }
+    let(:section) { FactoryBot.create(:section, article: article) }
+    let(:key) { FactoryBot.create(:key, project: project, section: section) }
+    let(:translation) { FactoryBot.create(:translation, key: key, copy: nil, source_copy: 'test', translator: nil, rfc5646_locale: 'fr-CA') }
+    let(:params) { ActionController::Parameters.new(translation: { copy: 'test copy', notes: 'test note' }) }
+    let(:mediator) { TranslationUpdateMediator.new(translation, reviewer, params) }
+
+    before do
+      article.update(ready: article_ready)
```

Comment:
I would add `article.update(ready: true)` in the test blocks where it is used. I think it would be mode clear to read.